bokehjs/src/coffee/api/typings/models/annotations.d.ts

```diff
@@ -54,6 +54,96 @@ declare namespace Bokeh {
         legends?: Array<[string, Array<GlyphRenderer>]>;
     }
 
+    export var ColorBar: { new(attributes?: IColorBar, options?: ModelOpts): ColorBar };
+    export interface ColorBar extends Annotation, IColorBar {}
+    export interface IColorBar extends IAnnotation {
+        location?: LegendLocation;
+        orientation?: Orientation;
+
+        legend_height?: Int;
+        legend_width?: Int;
```

Comment:
Why color bar has `legend_*` properties? I thought this was some WIP stuff in the original PR.